docs/src/views/ThemingExamples.tsx

```diff
@@ -121,32 +121,46 @@ export default () => (
         import { Button } from '@stardust-ui/react'
 
         const btnExample = () => (
-          <Button
-            icon={{
-              name: "at",
-              variables: { color: "brown" }
-            }}
-            content="Send email"
-            variables={{
-              secondaryColor: "coral",
-              secondaryBackgroundColor: "charcoal",
-              paddingLeftRightValue: 20,
-            }}
-          />
+        <Button
```

Comment:
unfortunately, this leads to change in the example's semantics. Originally the purpose was to demo `variables` usage - specifically, how it is possible to custoimize variables and obtain different look of the component. Now what we have is the `styles` demo, with additional aspect of using `variables` as style parameters.

Reply:
Talked to Roman about it - we should advertise the colors from the color palette rather than the usage of the styles.

Reply:
@alinais do you mean, for example, `siteVars.colors.pink[50]` instead of, say, `pink`? I have updated the PR, please let me know whether you like it now :)